### PR TITLE
[Admin] Footer

### PR DESCRIPTION
**Pull request by Rafikooo**
| Q               | A
|-----------------|-----
| Branch?         | 2.0 
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no 
| License         | MIT

image

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/views/Shared/Crud/Common/Content/footer.html.twig [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                        All rights reserved.
+                    </li>
+                </ul>
+            </div>
+
+            <div class="col-lg-12 ms-lg-auto">
+                <ul class="list-inline list-inline-dots m-1">
+                    <li class="list-inline-item">
+                        <a href="https://docs.sylius.com/en/1.13" target="_blank" class="link-secondary" rel="noopener">
+                            Stable Documentation
```
Comment (Rafikooo) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Shared/Crud/Common/Content/footer.html.twig]: I wasn't sure if we should also include translations for these new texts, so I decided to leave it for comments.

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "footer"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

File: src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/common/create.yaml [hunk 2]
```diff
@@ -17,6 +17,9 @@ twig_hooks:
                 template: '@SyliusAdmin/shared/crud/create/content/form.html.twig'
                 configuration:
                     render_rest: true
+            footer:
+                template: '@SyliusAdmin/shared/crud/common/content/footer.html.twig'
+                priority: -100
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/common/create.yaml:22]: We will add priorities to all the hooks at the end of the work on the BAP

File: src/Sylius/Bundle/AdminBundle/templates/shared/crud/common/content/footer.html.twig [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<footer class="footer footer-transparent d-print-none">
+    <div class="container-xl">
+        <div class="row text-center align-items-center">
+            <div class="col-12 col-lg-12">
+                <ul class="list-inline list-inline-dots">
+                    <li class="list-inline-item">
+                        Copyright © 2024
+                        <a href="https://sylius.com/" target="_blank" class="link-secondary">Sylius</a>.
+                        All rights reserved.
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/templates/shared/crud/common/content/footer.html.twig]: We should add all texts as translations

File: src/Sylius/Bundle/AdminBundle/templates/shared/crud/common/content/footer.html.twig [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+                    </li>
+                </ul>
+            </div>
+
+            <div class="col-lg-12 ms-lg-auto">
+                <ul class="list-inline list-inline-dots m-1">
+                    <li class="list-inline-item">
+                        <a href="https://docs.sylius.com/en/1.13" target="_blank" class="link-secondary" rel="noopener">
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/templates/shared/crud/common/content/footer.html.twig]:
```suggestion
                        <a href="https://docs.sylius.com" target="_blank" class="link-secondary" rel="noopener">
```

File: src/Sylius/Bundle/AdminBundle/templates/shared/crud/common/content/footer.html.twig [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+                            Stable Documentation
+                        </a></li>
+                    <li class="list-inline-item">
+                        <a href="https://new-docs.sylius.com" target="_blank" class="link-secondary" rel="noopener">
+                            New Documentation (Work in Progress)
+                        </a>
+                    </li>
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/templates/shared/crud/common/content/footer.html.twig]:
```suggestion
```
I would be for not referring to this documentation for now

File: src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/common/create.yaml (hunk 2 above)
Comment (Rafikooo) [reply src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/common/create.yaml:22]: There seems to be a problem with the layout in the following views. I wasn't able to set the footer at the bottom without using priorities.

image
image

File: src/Sylius/Bundle/AdminBundle/templates/shared/crud/common/content/footer.html.twig [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+                    <li class="list-inline-item">
+                        <a href="https://github.com/Sylius/Sylius/issues" target="_blank" class="link-secondary"
+                           rel="noopener">
+                            {{ 'sylius.ui.see_issue'|trans }}? {{ 'sylius.ui.report_it'|trans }}!
+                        </a>
+                    </li>
+                    <li class="list-inline-item">
+                        {{ 'sylius.ui.powered_by'|trans }} Sylius v{{ sylius_meta.version }}.
+                    </li>
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/templates/shared/crud/common/content/footer.html.twig:40]: I'm not sure if it wouldn't be overengineered, but this place seems perfect to trigger another hook 🤔

**Review by CoderMaggie — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Pull request merged**